app/controllers/fhir/r4/api_controller.rb

```diff
@@ -1044,6 +1067,10 @@ def search_patients(options)
         query = query.where('primary_telephone like ?', Phonelib.parse(search, 'US').full_e164)
       when 'email'
         query = query.where('email like ?', "%#{search}%")
+      when 'state-local-id'
+        query = query.where('lower(user_defined_id_statelocal) like ?', "#{search&.downcase}%")
+      when 'birthDate'
```

Comment:
Here FHIR wants this to be `birthdate`, all lowercase.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1098/commits/b335e9bae20603fd40a4b1f465c206aff9f4b9ef#diff-e79e38176ee697a8541cffba23c60d9decaa69382ca5c6e875da033508e5b428R346